src/bitcaster/models/notification.py

```diff
@@ -108,6 +112,9 @@ def notify_to_channel(self, channel: "Channel", assignment: Assignment, context:
             )
             dispatcher.send(addr.value, payload)
             return addr.value
+        else:
```

Comment:
remove this. It can produce a lot of warnings. Silent Events are already notified. This check can be moved  to Event/Channel check function (es. create an action in. the admin as 'Validate events')